app/constants.py

```diff
@@ -1 +1,31 @@
 API_VERSION = "v0.5.0beta"
+
+available_app_ids = {
+    "bos",
+    "btc-rpc-explorer",
+    "btcpayserver",
+    "chantools",
+    "circuitbreaker",
+    "electrs",
+    "faraday",
+    "fullynoded",
+    "joinmarket",
+    "kindle-display",
+    "lit",
+    "lnbits",
+    "lndconnect",
+    "lndmanage",
+    "loop",
+    "mempool",
+    "pool",
+    "pyblock",
+    "rtl",
+    "specter",
+    "sphinxrelay",
+    "stacking-sats-kraken",
+    "suez",
+    "telegraf",
+    "thunderhub",
+    "whitepaper",
+    "zerotier"
```

Comment:
I don't think we should make all apps available in the first step. 
I know for example when you activate ZeroTier, the SSH GUI asks for the ZT network to join. We need to find a mechanism to pass this data.

I think the client should know upfront which data is required and should submit it with the POST request upon activation.

@rootzoll what do you think?

Reply:
I used that way cause the LGTM bot complained that I passed in the data directly.